contracts/SBCWrapper.sol

```diff
@@ -46,7 +47,7 @@ contract SBCWrapper is IERC677Receiver, PausableEIP1967Admin, Claimable, Reentra
         tokenStatus[_token] = TokenStatus.ENABLED;
```

Comment:
Does it make sense to preserve the `UpdateSwapRate` event and emit it in both cases when `enableToken` is called to enable a token and when it is only necessary to update the rate?
So, there could be two sets of events emitted by `enableToken`:
1) the token is in the `DISABLED` status:
    * TokenSwapEnabled
    * UpdateSwapRate
2) the token is in the `ENABLED` status:
    * UpdateSwapRate

Reply:
I don't think it is beneficial to have two events instead of one. There was a small confusion in the event name, which we fixed. But apart from that, second event will only add a small overhead.

It could make sense to keep a separate event, if we expect a lot of these events to be generated, but that's clearly not the case here.